.ci/update-github-page.sh

```diff
@@ -67,9 +69,13 @@ java -jar contribution/releasenotes-builder/target/releasenotes-builder-1.0-all.
 echo ==============================================
 echo "GITHUB PAGE:"
 echo ==============================================
-CONTENT=$(cat github_post.txt)
+CONTENT=$(cat "$FILE")
 echo "$CONTENT"
 
+if [[ -n $(grep "@" "$FILE" ) ]]; then
+  sed -i "s/@/"'`@`'"/g" "$FILE"
```

Comment:
we need to escape content in JAVA code of releasenotes builder, to generate content that is ready for use without any hack after generation.

Reply:
Okay, I am looking into it!

Reply:
example of escaping in XML:
```
 <p>Notes:</p>
        <ul>
          <li>
            Github action &quot;Publish release notes outside&quot;.
            Author: github-actions[bot]
            <a href="https://github.com/checkstyle/checkstyle/issues/12082">#12082</a>
          </li>
          <li>
```
see it at https://cloud.drone.io/checkstyle/checkstyle/12325/4/2